src/a03-injection/step3.test.js

```diff
@@ -0,0 +1,39 @@
+import t from 'tap'
```

Comment:
I think naming here is a little bit vague, it would be better to use more suggestive names for variables, especially if this is a training/tutorial

Reply:
I've followed the code style from previous tests and other workshops such as [the-fastify-workshop](https://github.com/nearform/the-fastify-workshop)

Reply:
I based it on how it was done in the fastify workshop when I made the repo. No objections to changing it, but if we do we should change it everywhere.

I don't think it matters much either way though, it's a unit test so the structure is pretty obvious